tests/rptest/services/redpanda.py

```diff
@@ -868,83 +940,109 @@ def set_cluster_config(self,
     def set_resource_settings(self, rs):
         self._resource_settings = rs
 
+    @property
+    def si_settings(self):
+        return self._si_settings
 
-class RedpandaService(RedpandaServiceBase):
-    PERSISTENT_ROOT = "/var/lib/redpanda"
-    DATA_DIR = os.path.join(PERSISTENT_ROOT, "data")
-    NODE_CONFIG_FILE = "/etc/redpanda/redpanda.yaml"
-    CLUSTER_BOOTSTRAP_CONFIG_FILE = "/etc/redpanda/.bootstrap.yaml"
-    TLS_SERVER_KEY_FILE = "/etc/redpanda/server.key"
-    TLS_SERVER_CRT_FILE = "/etc/redpanda/server.crt"
-    TLS_CA_CRT_FILE = "/etc/redpanda/ca.crt"
-    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "redpanda.log")
-    BACKTRACE_CAPTURE = os.path.join(PERSISTENT_ROOT, "redpanda_backtrace.log")
-    COVERAGE_PROFRAW_CAPTURE = os.path.join(PERSISTENT_ROOT,
-                                            "redpanda.profraw")
+    def trim_logs(self):
+        if not self._trim_logs:
+            return
 
-    DEFAULT_NODE_READY_TIMEOUT_SEC = 20
+        # Excessive logging may cause disks to fill up quickly.
+        # Call this method to removes TRACE and DEBUG log lines from redpanda logs
+        # Ensure this is only done on tests that have passed
+        def prune(node):
+            node.account.ssh(
+                f"sed -i -E -e '/TRACE|DEBUG/d' {RedpandaService.STDOUT_STDERR_CAPTURE} || true"
+            )
 
-    DEDICATED_NODE_KEY = "dedicated_nodes"
+        self._for_nodes(self.nodes, prune, parallel=True)
 
-    RAISE_ON_ERRORS_KEY = "raise_on_error"
 
-    TRIM_LOGS_KEY = "trim_logs"
+class RedpandaServiceK8s(RedpandaServiceBase):
+    def __init__(self, context, num_brokers):
+        super(RedpandaServiceK8s, self).__init__(context, num_brokers)
+        self._helm = HelmTool(self)
+        self._kubectl = KubectlTool(self)
 
-    LOG_LEVEL_KEY = "redpanda_log_level"
-    DEFAULT_LOG_LEVEL = "info"
+    def start_node(self, node, **kwargs):
+        """
+        Install the helm chart which will launch the entire cluster. If
+        the cluster is already running, then noop. This function will not
+        return until redpanda appears to have started successfully. If
+        redpanda does not start within a timeout period the service will
+        fail to start.
+        """
+        self._helm.install()
```

Comment:
override Service::start?

Reply:
> override Service::start?
for now i'm sticking with the override of just the methods identified in the PR https://github.com/redpanda-data/redpanda/pull/9673 just to keep the scope to getting high-thoughput tier tests running, so i've skipped that method for now.